components/Auth/AuthContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import { Text, VStack } from '@chakra-ui/react'
+import React, { useState } from 'react'
+import Login from './Login'
+import SignUp from './Signup'
+
+interface AuthContainerProps {
+  initialState: 'sign_up' | 'log_in'
```

Comment:
Nit: make this a `type` like `type AuthState = 'sign_up' | 'log_in'` (Maybe something we can import from the `firebase` package?